thumbnail/index.php

```diff
@@ -5,27 +5,119 @@
 use Directus\Util\ArrayUtils;
 use Directus\Filesystem\Thumbnailer as ThumbnailerService;
 
-try {
+try 
+{
     $app = \Directus\Bootstrap::get('app');
-
-    // if the thumb already exists, return it
+    $config = $app->container->get('config')->get('thumbnailer');
+    
+    /*
+	    Set the fallback option in the thumbnailer config - '/api/configuration.php'
+	    If not set, a 404 img will be sent back if the dimensions 
+        are not supported within 'supportedThumbnailDimensions'
+    */
+    
+    if (ArrayUtils::get($config, 'fallback'))
+    {
+		/*
+			Fall back to one of supported sizes
+		*/
+	    
+	    $pathinfo = $app->request->getPathInfo();
+	    $pathinfo = ltrim($pathinfo, '/');
+	    $pathinfo = explode('/', $pathinfo);
+	    
+	    $config = $app->container->get('config')->get('thumbnailer');
+	    $sizes = ArrayUtils::get($config, 'supportedThumbnailDimensions');
+	    $redirect = NULL;
+	    
+	    /*
+		    Get the nearest size above the required size
+		    We must assume the first and second indexes are the width and height
+	    */
+	    
+	    foreach ($sizes as $size)
+	    {
```

Comment:
no a big deal, but we don't have this officially set but have been sticking to curly brackets on the same line, unless it's a function or method.

Reply:
oh okay no worries